### ISSUE TITLE
Exported image file's name is too long

### ISSUE THREAD

**Issue by moriyas1984** [shd101wyy/markdown-preview-enhanced#485]
I always use the feature of "Save as Markdown" to share a nice markdown rendered by the nice package, thanks.

But I think the file name of exported images is too long. They are created with such a name as "7fc86300e0ab6fd86f96d71ac69082309bd116a41a90d4f91c107efa789a1aa6ac3ff9c48e3385052e6af8e95b7cc11d1b199b13de12de51319022def0c8096a7982c29329e2a4f21a3cccd95cba23ed5af59152043932d0b534ca36c3ba2e88990", it has 196 characters.

Windows has a problem of the limit about the path length. It returns an error when it use a file has more than 256 characters path.

I think it is better to enable the user to specify file names or the length of them when the package exports converted images.

Please let me know how you think about it.

Thanks.

**Comment by shd101wyy** [comment]
Hi @moriyas1984 . Thanks for your advice. I will fix it in next version (`0.14.1`).

**Comment by shd101wyy** [comment]
Hi @moriyas1984 , is this issue fixed in the newest version?

**Comment by moriyas1984** [comment]
Hi @shd101wyy , thanks for your response to my request.
But I face difficulty in exporting images by "Save as Markdown" with the newest version (0.14.3).

My situation:
* Markdown preview is normally working and I right-click to export the file to a built-markdown.
* The built-markdown and image folder are created but images are not.
* In the built-markdown, I found a note such as follows where an image path should be inserted. 
```
ImageMagick is required to be installed to convert svg to png.
Error: spawn magick ENOENT
```  

Did you adopt a new method or application to export images?
Images had been normally exported in my environment before updating.

I updated Atom (1.14.2 -> 1.18) and the package (0.12 -> 0.14) yesterday.
I couldn't find a solution in the repository and documents...
Do you have any idea?

Thanks.

**Comment by shd101wyy** [comment]
@moriyas1984 Yes, you need to install [ImageMagick](http://www.imagemagick.org/script/download.php) separately to convert svg image to png image.

**Comment by moriyas1984** [comment]
@shd101wyy Thanks for your advice.
I have just exported a markdown containing a code-block of plantUML, it works.
The png file was normally exported with significantly short name :)

I appreciate your continuous efforts, thank you:)

**Comment by shd101wyy** [comment]
@moriyas1984 The next version will support `filename` property.  
For example, for the code <code>\`\`\`plantuml {filename="my_puml.png"}</code>, when you save as markdown, the image file will be output to `${imageFolder}/my_puml.png`.
